flutter_keyboard_visibility_web/lib/flutter_keyboard_visibility_web.dart

```diff
@@ -1,25 +1,46 @@
-import 'dart:html' as html show window, Navigator;
+import 'dart:async';
+import 'package:web/web.dart';
 import 'package:flutter_keyboard_visibility_platform_interface/flutter_keyboard_visibility_platform_interface.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
 /// The web implementation of the [FlutterKeyboardVisibilityPlatform] of the
 /// FlutterKeyboardVisibility plugin.
-class FlutterKeyboardVisibilityPluginWeb
-    extends FlutterKeyboardVisibilityPlatform {
+class FlutterKeyboardVisibilityPluginWeb extends FlutterKeyboardVisibilityPlatform {
   /// Constructs a [FlutterKeyboardVisibilityPluginWeb].
-  FlutterKeyboardVisibilityPluginWeb(html.Navigator navigator);
+  FlutterKeyboardVisibilityPluginWeb(Navigator navigator);
+
+  static final _onChangeController = StreamController<bool>();
+  static final _onChange = _onChangeController.stream.asBroadcastStream();
 
   /// Factory method that initializes the FlutterKeyboardVisibility plugin
   /// platform with an instance of the plugin for the web.
   static void registerWith(Registrar registrar) {
-    FlutterKeyboardVisibilityPlatform.instance =
-        FlutterKeyboardVisibilityPluginWeb(html.window.navigator);
+    FlutterKeyboardVisibilityPlatform.instance = FlutterKeyboardVisibilityPluginWeb(window.navigator);
+    EventStreamProvider<Event>('resize').forTarget(window.visualViewport).listen((e) {
+      final minKeyboardHeight = 200;
```

Comment:
Can you please explain this constant value of `200`?

Reply:
it could be something like 180 and constant was put 200 as an estimate

look at this article
https://federicabenacquista.medium.com/list-of-the-official-ios-keyboards-heights-and-how-to-calculate-them-c2b844ef54b9

Reply:
On my opinion an medium article (also an article that costs) is not the right source to implement something on. Keyboards on my point of view are dynamically (specially with accessibility/additional features) in height and width.

But I'll mark this as resolved because I'm not related into this topic and feel free to contribute :).